### DISCUSSION TITLE
Baro And Vnav

### DISCUSSION THREAD

**Discussion by RubiXKube** [calibx/msfsdeck#165] [Ideas]
Is it possible to add a function when pushing baro encoder ?
There is two choice, reset it to 29.92
Or use "standart barometric" for liners and set it in STD ?

**Comment by RubiXKube** [comment]
Like to see a VNAV toggle button to
Simconnect event seems to be AP_BC_HOLD
usefull for airliner or little jet 
May be in Airliner folder and separate command ?
Thanks !!

**Comment by calibx** [comment]
Will add it soon

**Comment by calibx** [comment]
AP_BC_HOLD is to activate back course on the AP I don't think it is the VNAV one.
VNAV seems to be another L:VAR ou H:VAR ... needing the WASM module

**Comment by RubiXKube** [comment]
You are probably right, i said that because i'm using Honda Jet
https://docs.google.com/document/d/1FnDKlKeVm8yFkKjqKo4ySdSDU0pOZ1AoLAXcIIOBelQ/edit

**Comment by calibx** [comment]
Oh ok I can add the toogle and it will work ! I didn't know they used it for VNAV :)

**Comment by calibx** [comment]
Coding done, will be included in the next 2.3 release

**Reply by RubiXKube** [reply]
Thanks !
